R/gemtc_analysis.R

```diff
@@ -160,16 +164,21 @@ CovariateModelOutput <- function(model, cov_value) {
   names(intercepts) <- levels(model$model$data$reg.control)[! levels(model$model$data$reg.control) %in% model$model$data$reg.control]
   
   # naming conventions to match current Bayesian functions
-  return(list(
-    mtcResults = model,
-    mtcRelEffects = rel_eff,
-    rel_eff_tbl = rel_eff_tbl,
-    a = model_text,
-    sumresults = summary,
-    dic = fit_stats,
-    cov_value_sentence = cov_value_sentence,
-    slopes = slopes,
-    intercepts = intercepts)
+  return(
+    list(
+      mtcResults = model,
+      mtcRelEffects = rel_eff,
+      rel_eff_tbl = rel_eff_tbl,
+      a = model_text,
+      sumresults = summary,
+      dic = fit_stats,
+      cov_value_sentence = cov_value_sentence,
+      slopes = slopes,
+      intercepts = intercepts,
+      outcome = outcome_measure,
+      mtcNetwork = model$model$network,
+      model = model$model$linearModel
```

Comment:
I agree, perhaps the parameter name of the function should be changed, perhaps to `model_fit`

Reply:
I agree that this needs changing, but to do so here would cause some nasty conflicts with another branch I'm working on. This is because I'd also need to change the `baye()` function in `bayes_analysis.R` to match this new name, and everywhere which uses the model output. If you're happy to defer it, I'll raise it as a separate Jira task for after things have calmed down.